posixlib/src/main/scala/scala/scalanative/posix/sys/wait.scala

```diff
@@ -0,0 +1,43 @@
+package scala.scalanative.posix.sys
+
+import scala.scalanative.unsafe._
+import scala.scalanative.posix.sys.types.pid_t
+
+@extern
+object wait {
+  @name("scala_native_wait_WCONTINUED")
+  val WCONTINUED: CInt = extern
+
+  @name("scala_native_wait_WNOHANG")
+  val WNOHANG: CInt = extern
+
+  @name("scala_native_wait_WUNTRACED")
+  val WUNTRACED: CInt = extern
+
+  @name("scala_native_wait_WIFEXITED")
+  def WIFEXITED(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WEXITSTATUS")
+  def WEXITSTATUS(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WIFSIGNALED")
+  def WIFSIGNALED(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WTERMSIG")
+  def WTERMSIG(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WIFSTOPPED")
+  def WIFSTOPPED(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WSTOPSIG")
+  def WSTOPSIG(statVal: CInt): CInt = extern
+
+  @name("scala_native_wait_WIFCONTINUED")
+  def WIFCONTINUED(statVal: CInt): CInt = extern
+
+  //renamed to avoid shadowing by with Object.wait
+  @name("wait")
+  def waitpid(statLoc: Ptr[CInt]): pid_t = extern
+
```

Comment:
Note to other reviewers, other readers, and self:
`int    waitid(idtype_t, id_t, siginfo_t *, int);` from the 
2018 spec, which I think is the pertinent one here, is missing
at this point.  I believe that Mark is working on the (complicated)
`signal.h` required to implement it.

Reply:
I missed it somehow, but I'll add it in my next PR

Reply:
In the interest of moving onward.

Given that `waitid()` is blocked on the implementation of `signal.h` and 
implementing the latter is bigger than a breadbox, I propose creating
an Issue stating that `waitid()` needs to be completed when `signal.h` becomes available. This should keep the work item someplace where it will be seen from time to time and not fade from memory as people change.

Then in the code I would put comments such as:
```
// Not implemented, requires nonextant signal.h.  See Issue #???
// int    waitid(idtype_t, id_t, siginfo_t *, int);
```
What do you think?  Knock the little devil dead, or at least stunned?

This may the best of a bad lot.  Scala Center reviewers may decline this solution because they prefer no commented out code.  Let's see what
flies.